packages/styles/src/components/forms.scss

```diff
@@ -51,25 +51,90 @@ select.form-control-rg:not([size]):not([multiple]) {
       border-left: inherit;
     }
 
+    // Fix Safari cutoff when change the initial position of the selector button.
+    @supports (background: -webkit-named-image(i)) {
+      &::file-selector-button {
+        display: initial;
+        position: initial;
+        margin-inline: 0;
+        padding-inline: 0;
+        width: 0;
+        border: 0 none;
+      }
+
+      &::after {
+        display: flex;
+        align-items: center;
+        content: 'Choose File';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        z-index: 2;
+        padding-inline: forms.$input-padding-y;
+        background-color: inherit;
+        border-left: inherit;
+        text-align: center;
+
+        [lang='de'] & {
+          content: 'Datei auswählen';
+        }
+
+        [lang='fr'] & {
+          content: 'Choisir un fichier';
+        }
+
+        [lang='it'] & {
+          content: 'Scegli file';
+        }
+      }
+
+      &[multiple] {
+        &::after {
+          content: 'Choose Files';
+
+          [lang='de'] & {
+            content: 'Dateien auswählen';
+          }
+
+          [lang='fr'] & {
+            content: 'Sélect. fichiers';
```

Comment:
Not sure if we should shorten texts. Would it be "Sélectionner des fichiers" or "Choisir des fichiers"? I get that it's quite long. This article recommends to not use abbreviations https://www.experience-hub.ch/document/2777#/schreiben-digitaler-inhalte/einfache-sprache. What do you think?

Reply:
Good that we have a best practice. I think you are totally right, both look good to me, so I'll take the short version.